src/pages/FindPW/FindPW.scss

```
@@ -0,0 +1,40 @@
+.findPWFrame {
+  height: 80vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  .findpwText {
+    font-size: 25px;
+    color: rgb(133, 133, 133);
+    margin-bottom: 40px;
+    font-weight: 600;
+  }
+  .formPWFrame {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    .emailInput {
+      width: 350px;
+      padding: 15px;
+      border: none;
+      border-bottom: 1px solid gray;
+    }
+  }
+  .confirButtonFrame {
+    display: flex;
```

별 것 아닌것 같지만 컨벤션을 지키며 코딩하는것은 생각보다 중요합니다. 선택자와 직전 선택자의 스타일링 코드 사이에는 빈 여백으로 가독성을 증가시켜주세요

```suggestion
  align-items: center;
  
  .findpwText {
    font-size: 25px;
    color: rgb(133, 133, 133);
    margin-bottom: 40px;
    font-weight: 600;
  }
  
  .formPWFrame {
    display: flex;
    justify-content: center;
    align-items: center;
    
    .emailInput {
      width: 350px;
      padding: 15px;
      border: none;
      border-bottom: 1px solid gray;
    }
  }
  
  .confirButtonFrame {
    display: flex;
```

---

넵 수정후 적용 완료했습니다.